_unused_topics/microshift-nodeport-unreachable-workaround.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * module may be unused in 4.13
+
+:_content-type: PROCEDURE
+[id="microshift-nodeport-unreachable-workaround_{context}"]
+= Manually restarting the `ovnkube-master` pod to resume node port traffic
+
+After you install {product-title}, NodePort service traffic might stop. To troubleshoot this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* A cluster installed on infrastructure configured with the Open Virtual Network (OVN)-Kubernetes network plugin.
+* Access to the `kubeconfig` file.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+Run the commands listed in each step that follows to restore the `NodePort` service traffic after you install{product-title}:
```

Comment:
Per the Modular documentation guidelines, the "Procedure" section should only contain the steps. You can remove this sentence which repeats what is in the procedure introduction.